docs/advanced/Testing.md

```diff
@@ -1,9 +1,35 @@
 # Examples of Testing Sagas
 
-**WIP**
+There is ongoing discussion in [this issue](https://github.com/redux-saga/redux-saga/issues/518) about the best way to test Sagas.
 
-See Repository Examples:
+
+In the repo we also have two examples showing how Sagas can be tested:
 
 https://github.com/redux-saga/redux-saga/blob/master/examples/counter/test/sagas.js
 
 https://github.com/redux-saga/redux-saga/blob/master/examples/shopping-cart/test/sagas.js
+
+
+
+***External Resources***
+
+Redux Saga Test Plan - https://github.com/jfairbank/redux-saga-test-plan
+
+From the README:
```

Comment:
While I would be flattered to have redux-saga-test-plan listed in the docs, I'm not sure it would be fair to some of the other redux-saga testing libraries out there. Only listing one or two would give the impression that redux-saga favors those over others. This is part of the reason why I've never even made a PR to add redux-saga-test-plan to the docs.

I'm deeply interested in the redux-saga testing story and initially only embraced unit testing with redux-saga-test-plan. However, I have come to recognize that integration testing is probably a better approach which is why redux-saga-test-plan now offers it with a simple, declarative API.

When redux-saga's API settles, I would love to be involved in the testing discussion further. For example, features like #586 could be useful for simplifying some of the hacks I need to use currently or for future features in redux-saga-test-plan.

I'm still okay providing a description for this PR if you want, but it would only be fair to list other testing libraries like wix's too.